api/src/main/java/org/openmrs/module/bedmanagement/BedLayoutWithDetails.java

```diff
@@ -1,80 +0,0 @@
-package org.openmrs.module.bedmanagement;
-
-import org.openmrs.module.bedmanagement.entity.Bed;
-import org.openmrs.module.bedmanagement.entity.BedPatientAssignment;
-
-import java.util.Set;
-
-public class BedLayoutWithDetails {
```

Comment:
So this "details" class was actually just a subset of BedLayout?  Not what I would have expected, and, yeah, makes sense to remove!

Reply:
Yes. It was only really used as a means to use a Hibernate Bean Transformer to read the results of an HQL Query directly into a bean.  I've changed that Hibernate method to not use this, so it could go away.